config/helm/chart/default/templates/Common/webhook/deployment-webhook.yaml

```diff
@@ -116,4 +116,10 @@ spec:
       imagePullSecrets:
         - name: {{ .Values.operator.customPullSecret }}
       {{- end }}
+      {{- if .Values.operator.nodeSelector }}
+      nodeSelector: {{- toYaml .Values.operator.nodeSelector | nindent 8 }}
+      {{- end }}
+      {{- if .Values.operator.tolerations }}
+      tolerations: {{- toYaml .Values.operator.tolerations | nindent 8 }}
+      {{- end -}}
```

Comment:
the values should come from the webhook section in the values file instead of the operator one so that they are clearly separated.